### DISCUSSION TITLE
Cities

### DISCUSSION THREAD

**Discussion by TheMowgliMan** [TheMowgliMan/Country-Clash#19] [Announcements]
Let me know what to improve.
![Country Clash- The Game (DEBUG) 5_25_2022 11_50_43 AM](https://user-images.githubusercontent.com/96443213/170346389-ffbfa494-71cd-4d53-80cd-f9ca6721fb51.png)

**Comment by 3quarterscolon** [comment]
Nice. I think that there should be more cities, though. Maybe some should only appear when zoomed in further based on their size?

**Comment by TheMowgliMan** [comment]
Good idea. It generates a city's pop, but doesn't store it.

**Comment by TheMowgliMan** [comment]
Creating an issue from this.
